Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -20,16 +20,17 @@ For installing ``kubeadm`` and for more provisioning options please refer to
 .. note::
 
    Cilium's kube-proxy replacement depends on the :ref:`host-services` feature,
-   therefore a v4.19.57, v5.1.16, v5.2.0 or more recent Linux kernel is required.
+   therefore a v4.19.57, v5.1.16, v5.2.0 or more recent Linux kernel is required!
```

Comment:
Nit: documentation should generally be written in neutral style.

```suggestion
   therefore a v4.19.57, v5.1.16, v5.2.0 or more recent Linux kernel is required.
```

Reply:
Makes sense, will add in a new PR as discussed offline.